Gemfile.lock

```diff
@@ -268,6 +268,7 @@ GEM
     faraday-net_http (3.4.0)
       net-http (>= 0.5.0)
     ffi (1.17.1-arm64-darwin)
+    ffi (1.17.1-x86_64-darwin)
```

Comment:
You'll need to remove the commit which adds these changes, as it will break the pipeline again 😁